scripts/generate_constraint_tmpl.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
```

Comment:
It looks like the makefile is calling gomplate directly. Is this script still used?

Reply:
This is a script that helped me reverse generate constraint.tmpl from existing template.yaml. I don't think we need it anymore.